test/test.js

```diff
@@ -0,0 +1,47 @@
+import assert from "assert";
+import fs from "fs";
+
+import { unified } from "unified";
+import remarkParse from "remark-parse";
+import remarkStringify from "remark-stringify";
+import presetLintNode from "../index.js";
+import { read } from "to-vfile";
+import { reporter } from "vfile-reporter";
+
+const linter = unified()
+  .use(remarkParse)
+  .use(presetLintNode)
+  .use(remarkStringify);
+
+// Test that correctly-formatted markdown is ok.
+(async () => {
+  const file = await read(new URL("./fixtures/ok.md", import.meta.url));
+  const result = await linter.process(file);
+  assert.strictEqual(result.messages.length, 0, reporter(result));
+})().catch((e) => {
+  console.error(e);
+  process.exit(1);
+});
+
+// Test that incorrectly-formatted markdown fails.
+(async () => {
+  const file = await read(new URL("./fixtures/fail.md", import.meta.url));
```

Comment:
Should the file be a little more descriptive and/or follow a pattern? Ex: `bad-title.md`, `bad-mailto.md`, etc...

Reply:
For now, there's only one "make sure it says an OK file is OK" and one "make sure it flags an error as an error". They aren't really checking specific rules or anything. So I'm fine with leaving them as `ok.md` and `fail.md` until we have more of these files. But I also don't feel strongly about it. I just don't know what to change these two files' names to, though.

Reply:
If I'm adding fixtures for YAML in a follow-up PR, we soon will have several test files supposed to fail.

Reply:
I'm OK if we want to "solve" the naming in that future PR